components/suggest/src/schema.rs

```diff
@@ -170,6 +169,22 @@ CREATE TABLE fakespot_custom_details(
     total_reviews INTEGER NOT NULL,
     FOREIGN KEY(suggestion_id) REFERENCES suggestions(id) ON DELETE CASCADE
 );
+CREATE VIRTUAL TABLE IF NOT EXISTS fakespot_fts USING FTS5(
+  title,
+  prefix='4 5 6 7 8 9 10 11',
+  content='',
+  tokenize=\"porter unicode61 remove_diacritics 2 tokenchars '''-'\"
```

Comment:
`'''-'\"` reads a bit cryptic, maybe we can add a comment clarifying that both apostrophe and hyphen are treated as valid tokenchars by the tokenizer.